python_modules/libraries/dagster-tableau/dagster_tableau/assets.py

```diff
@@ -1,25 +1,37 @@
 from collections.abc import Sequence
 from typing import TYPE_CHECKING, Optional, cast
 
-from dagster import AssetExecutionContext, AssetsDefinition, AssetSpec, multi_asset
-from dagster._annotations import beta
+from dagster import (
+    AssetExecutionContext,
+    AssetsDefinition,
+    AssetSpec,
+    _check as check,
```

Comment:
nit: this has moved -- you can do `from dagster.shared import check` (_check is now just an alias for that)